packages/gatsby-theme-nodeschool/gatsby-node.js

```diff
@@ -83,3 +86,50 @@ exports.createPages = async ({ graphql, actions, reporter }/*, themeOptions*/) =
     });
   });
 };
+
+exports.onPostBootstrap = ({ store }) => {
+  const { program } = store.getState();
+  const publicLocales = path.join(program.directory, `public/locales`);
+  console.log(`Copying locales`);
+  fsExtra.copySync(path.join(__dirname, `/src/locales`), publicLocales);
+};
+
+exports.createSchemaCustomization = ({ actions }) => {
+  const { createTypes } = actions;
+  const typeDefs = `
```

Comment:
Oh cool, I was going to submit this upstream but you already did it. I would suggest making a pr for gatsby-source-meetup as well

Reply:
Hmmm, ok, I will make a PR to gatsby-source-meetup about it.

Reply:
Moved to a Pullrequest in gatsby-source-meetup:

- https://github.com/phacks/gatsby-source-meetup/pull/11